incubator/oauth-proxy/README.md

```diff
@@ -73,4 +75,21 @@ Alternatively, a YAML file that specifies the values for the above parameters ca
 $ helm install incubator/oauth-proxy --name my-release -f values.yaml
 ```
 
+### Configuration of htpasswd config file
+
+If you want to, in adition to oauth2, have some basic auth working, specify the `htpasswd` section.
+You can see an example of how to generate the contents of this file below.
+
+You will have to specify the `htpasswd-file` parameter pointing to the directory `/htpasswd/htpasswd` in the values.yaml:
+```
+extraArgs:
+  htpasswd-file: "/htpasswd/htpasswd"
+```
+
+You can generate a password file with:
+```
+htpasswd -sbc /tmp/pwfile username password
+```
+Which will add a username/password of `username/password` to the file `/tmp/pwfile`.
```

Comment:
Should we add a note for new helm users that the contents of this file then should be copied into the values.yaml?

Reply:
Sure thing. I added a bogus entry in the file `values.yaml`, I'll add a note.
Thanks.